docs/runtime/instantiable-module.md

```diff
@@ -0,0 +1,140 @@
+<Something<I>>::put(something); -->
+
+### Event initialization
+```rust
+fn deposit_event<T, I>() = default;
+```
+
+### Event Declaration
+```rust
+decl_event!(
+	pub enum Event<T, I> where AccountId = <T as system::Trait>::AccountId {
+		...
+	}
+}
+```
+
+## Deploying an Instantiable Module
+
+When composing a runtime, one must take care to use the proper syntax for instantiable modules. As before it is slightly different.
+
+### Implementing the Configuration Trait
+When implementing the module's configuration trait, you need to be explicit which instance you are configuring, because in general the instances need not have identical configurations. In this snippet we've named the instance `Instance1`.
+```rust
+impl template::Trait<template::Instance1> for Runtime {
+	type Event = Event;
+}
+```
+
+### In the `construct_runtime!` Macro
+Finally, in the `construct_runtime!` macro, we give a more human-friendly name to each instance of the module. Here I've called `Instance1` `FirstTemplate`.
+```rust
+FirstTemplate: template::<Instance1>::{Module, Call, Storage, Event<T>, Config},
+```
+
+The default Substrate node does a similar thing with its council and technical committee.
+```rust
+Council: collective::<Instance1>::{Module, Call, Storage, Origin<T>, Event<T>, Config<T>},
+TechnicalCommittee: collective::<Instance2>::{Module, Call, Storage, Origin<T>, Event<T>, Config<T>}
+```
+
+## Default Instance
+One drawback of instantiable modules, as we've presented them so far is that they require the deployer to use the more elaborate syntax even if they only desire a single instance of the module. To alleviate this inconvenience, Substrate provides a feature known as DefaultInstance. This allows runtime developers to deploy an instantiable module exactly as they would if it were not instantiable provided they **only use a single instance**.
+
+To make your instantiable module support DefaultInstance, you must specify it in three places.
+
+```rust
+pub trait Trait<I=DefaultInstance>: system::Trait {
+```
+
+```rust
+decl_storage! {
+  trait Store for Module<T: Trait<I>, I: Instance=DefaultInstance> as TemplateModule {
+    ...
+  }
+}
+```
+
+```rust
+decl_event!(
+	pub enum Event<T, I=DefaultInstance> where ... {
+    ...
+  }
+}
+```
+
+Having made these three changes, a developer who uses your module doesn't need to know or care that your module is instantable. They can deploy it just as they would any other module.
+
+## Genesis Configuration
+Some modules require a genesis configuration to be specified. This process is identical for instantiable and non-instantiable modules, but it is worth mentioning that the configuration is named after the human-friendly name given in `construct_runtime!`, and is different for each instance. That is to say that the configuration is **not** named after the module itself. Let's look to the default Substrate node's use of the Collective module as an example.
```

Comment:
"the configuration is **not** named after the module itself."

This is really not helpful. Just say what you want to say:

"When defining the genesis config for an instantiable module, you must append the "Instance Name" to the name of the module.

For example, the `TechnicalComittee` and `Council` in the main Substrate node both use the `collective` module:

```rust
Council: collective::<Instance1>::{Module, Call, Storage, Origin<T>, Event<T>, Config<T>},
TechnicalCommittee: collective::<Instance2>::{Module, Call, Storage, Origin<T>, Event<T>, Config<T>}
```

You would then configure these modules like so:
...

Reply:
I agree and think it is easier to read if you just cut the second to last sentence.

Reply:
I included this specifically because I got stuck on it while learning. I thought I would use `CollectiveConfig` twice as opposed to `CouncilConfig` and `TechnicalCommitteeConfig`.

However I appreciate the peer review and have taken the suggestion.